browser/src/Editor/OniEditor/OniEditor.tsx

```diff
@@ -145,7 +146,9 @@ export class OniEditor implements IEditor {
             wrapReactComponentWithLayer("oni.layer.errors", <ErrorsContainer />),
         )
 
-        this._neovimEditor.bufferLayers.addBufferLayer("image", buf => new ImageBufferLayer(buf))
+        this._neovimEditor.bufferLayers.addBufferLayer(buf => {
+            return [".png", ".jpg"].includes(path.extname(buf.filePath))
```

Comment:
If possible, it may be worth using the file types that are stored in the config : https://github.com/onivim/oni/blob/master/extensions/images/package.json

Then we'd only have 1 version of the truth, but I also don't know how easy it is to grab them values.

Reply:
@CrossR I'm not sure about pulling values out of one of the many packages.json in the oni editor but can copy the list across although one thing I notice is that it includes svgs which tbh I work with on occasion and I think unlike binary ones like png etc. svg is similar to xml so these should be editable in the editor not sure if the layer is toggleable but adding svgs there would make them as things currently are not possible to work with.

Reply:
I'd be OK ditching that extension for now and supplying this via a `config` setting, too. The only purpose that extension is set to the filetype, though, so if we go that route, we'd want to remove that extension also.